src/contexts/account/useWithdrawals.ts

```diff
@@ -38,27 +38,29 @@ export const useWithdrawalBot = (address: string | null) => {
   // }, [finalizeWithdrawal]);
 };
 
-export const useClaimingBot = (address: string | null) => {
-  const { kit } = useCelo();
+export const useClaimingBot = (address: `0x${string}` | undefined) => {
   const { api } = useAPI();
   const { accountContract } = useBlockchain();
+  const publicClient = usePublicClient();
+  const { refetch: loadPendingWithdrawals } = useContractRead({
+    ...accountContract,
+    functionName: address && 'getPendingWithdrawals',
```

Comment:
rather than the address && function name pattern i think it would be more idiomatic to use the `enabled` field https://wagmi.sh/react/hooks/useContractRead#enabled-optional to prevent running